docs/tutorials/00-landscaper-installation.md

```diff
@@ -0,0 +1,138 @@
+- A nginx ingress controller must be deployed in the target cluster
+- The command "htpasswd" must be installed on your local machine.
+
+First the helm values for the Landscaper configuration (`landscaper-values.yaml`) have to be created:
+
+```yaml
+landscaper:
+    landscaper:
+      registryConfig:
+        cache: {}
+        allowPlainHttpRegistries: false
+        insecureSkipVerify: false
+      
+      deployers: 
+      - container
+      - helm
+      - manifest
+
+      deployerManagement:
+        disable: false
+        namespace: ls-system
+        agent:
+          disable: false
+          namespace: ls-system
+```
+
+This tutorial assumes that the OCI registry usernames is store in environment variable `REGISTRY_USER` and the password is stored in environment variable `REGISTRY_PASSWORD`.
+
+To install the Landscaper and the OCI registry, use the following command:
+
+```shell script
+landscaper-cli quickstart install \
+    --kubeconfig ./kubeconfig.yaml \
+    --landscaper-values ./landscaper-values.yaml \
+    --namespace ls-system \
+    --install-oci-registry \
+    --install-registry-ingress \
+    --registry-username ${REGISTRY_USER} \
+    --registry-password ${REGISTRY_PASSWORD}
+```
+
+The Landscaper will automatically be configured with the provided OCI registry credentials.
+After a successful installation the `landscaper-cli` will print the URL of the OCI registry endpoint:
+
+```
+Landscaper installation succeeded!
+
+The OCI registry can be accessed via the URL https://o.ingress.cluster-domain
+It might take some minutes until the TLS certificate is created
+```
+
+To test the OCI registry is working, issue the following command:
+
+```shell script
+curl --location --request GET https://o.ingress.cluster-domain/v2/_catalog -u "${REGISTRY_USER}:${REGISTRY_PASSWORD}
+```
+
+This should give the output:
+
+```json
+{"repositories":[]}
+```
+
+#### Alternative 2: Install using an external OCI registry
+
+##### Prerequisites
+
+- An OCI registry that implements the [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec).
+
+First the helm values for the Landscaper configuration (`landscaper-values.yaml`) have to be created:
+
+```yaml
+landscaper:
+    landscaper:
+      registryConfig:
+        cache: {}
+        allowPlainHttpRegistries: false
+        insecureSkipVerify: false
+        secrets:
+          default: {
+            "auths": {
+              "hostname": {
+                "auth": "my auth"
+              }
+            }
+          }
+      
+      deployers: 
+      - container
+      - helm
+      - manifest
+
+      deployerManagement:
+        disable: false
+        namespace: ls-system
+        agent:
+          disable: false
+          namespace: ls-system
+```
+
+The registry secrets have to be provided in a plain Docker auth JSON format. See [here](../gettingstarted/install-landscaper-controller.md#Private registry access) for more details.
```

Comment:
The referenced documentation does not describe which section in `landscaper-values.yaml` to use for configuring the registry secrets. Here you find `landscaper.landscaper.registrySecrets` but this differs from `landscaper.landscaper.registryConfig.secrets`.

Reply:
It seems that the refernced documentation is wrong with `landscaper.landscaper.registrySecrets`.
`landscaper.landscaper.registryConfig.secrets`is correct.
I will fix the referenced doc in a separate PR.